docs/opcodes/55/berlin.mdx

```diff
@@ -1,5 +1,10 @@
 ## Gas
 
+### Definitions:
+- **value**: The value of the storage slot after executing the opcode SSTORE.
+- **current_value**: The value of the storage slot before executing the opcode SSTORE.
+- **original_value**: The value of the storage slot at the beginning of the transaction.
```

Comment:
I think it would be more clear to say "before the current transaction". (please modify in the "estimate" section too :-) )

Reply:
I changed it